_config.yml

```diff
@@ -122,3 +122,10 @@ compress_html:
   profile: false
   # ignore:
   #   envs: all
+
+# Enable support for mermaid, defaults to true
+mermaid_enabled: true
+# pick from https://cdnjs.com/libraries/mermaid
+mermaid_version: "9.0.0"
+# choices: https://mermaid-js.github.io/mermaid/#/theming
+mermaid_theme: "forest"
```

Comment:
```suggestion
mermaid_theme: "default"
```

I think the default theme compliments the site's existing color set better 👀 

# Before

<img width="249" alt="image" src="https://user-images.githubusercontent.com/60357436/169583232-ff17654b-90df-4a95-b947-8923bf7e5fed.png">

# After

<img width="210" alt="image" src="https://user-images.githubusercontent.com/60357436/169584005-2745852c-9d87-42ab-a342-4dacd681cf16.png">